packages/styles/src/mixins/_utilities.scss

```diff
@@ -176,3 +178,12 @@
     }
   }
 }
+
+@mixin responsive-actions {
```

Comment:
Can you move the [margin-left](https://github.com/swisspost/design-system/pull/3616/files#diff-2f8b0d15fcfb213824d105a0ac65edbd2aefe77692061fcf8d1585f1c0773b64R28) and [flex-direction](https://github.com/swisspost/design-system/pull/3616/files#diff-2f8b0d15fcfb213824d105a0ac65edbd2aefe77692061fcf8d1585f1c0773b64R34) to the mixin as well so that we do not have media queries in the component styles at all?

Reply:
I moved the margin-left to the mixin, but the flex-direction: row is outside of those two action buttons on the right

Reply:
You can refactor the mixin and apply it to the parent, I could imagine it being used in multiple components like modal, [notifications](https://www.figma.com/design/JIT5AdGYqv6bDRpfBPV8XR/Foundations-%26-Components-Next-Level?node-id=85-321&node-type=canvas&t=ia7Kw9j7TcqUAhOm-0), and [accordions](https://www.figma.com/design/xZ0IW0MJO0vnFicmrHiKaY/Components-Post?node-id=42-358&node-type=canvas&m=dev).

This way the &-right-actions` selector would be part of the mixin and the classes would be consistent for all components. What do you think?

Reply:
You're right, it makes more sense that way :)